Patch to revert the following upstream commit:

commit d896874f868987e22489c64efae5a0f62b070d47
Author: Arend van Spriel <arend.vanspriel@broadcom.com>
Date:   Mon Jan 7 12:14:41 2019 +0100

    nl80211: Indicate 802.1X 4-way handshake offload in connect
    
    Upon issuing a connect request we need to indicate that we want the
    driver to offload the 802.1X 4-way handshake for us. Indicate it if
    the driver capability supports the offload.
    
    Signed-off-by: Arend van Spriel <arend.vanspriel@broadcom.com>

diff --git b/src/drivers/driver_nl80211.c a/src/drivers/driver_nl80211.c
index a24497f..5081b5b 100644
--- b/src/drivers/driver_nl80211.c
+++ a/src/drivers/driver_nl80211.c
@@ -5568,13 +5568,6 @@ static int nl80211_connect_common(struct wpa_driver_nl80211_data *drv,
 			return -1;
 	}
 
-	if (params->req_key_mgmt_offload &&
-	    (drv->capa.flags & WPA_DRIVER_FLAGS_4WAY_HANDSHAKE_8021X)) {
-		    wpa_printf(MSG_DEBUG, "  * WANT_1X_4WAY_HS");
-		    if (nla_put_flag(msg, NL80211_ATTR_WANT_1X_4WAY_HS))
-			    return -1;
-	    }
-
 	/* Add PSK in case of 4-way handshake offload */
 	if (params->psk &&
 	    (drv->capa.flags & WPA_DRIVER_FLAGS_4WAY_HANDSHAKE_PSK)) {
diff --git b/wpa_supplicant/wpa_supplicant.c a/wpa_supplicant/wpa_supplicant.c
index e0ee553..93630e4 100644
--- b/wpa_supplicant/wpa_supplicant.c
+++ a/wpa_supplicant/wpa_supplicant.c
@@ -3114,13 +3114,6 @@ static void wpas_start_assoc_cb(struct wpa_radio_work *work, int deinit)
 			params.psk = ssid->psk;
 	}
 
-	if ((wpa_s->drv_flags & WPA_DRIVER_FLAGS_4WAY_HANDSHAKE_8021X) &&
-	    (params.key_mgmt_suite == WPA_KEY_MGMT_IEEE8021X ||
-	     params.key_mgmt_suite == WPA_KEY_MGMT_IEEE8021X_SHA256 ||
-	     params.key_mgmt_suite == WPA_KEY_MGMT_IEEE8021X_SUITE_B ||
-	     params.key_mgmt_suite == WPA_KEY_MGMT_IEEE8021X_SUITE_B_192))
-		params.req_key_mgmt_offload = 1;
-
 	if (wpa_s->conf->key_mgmt_offload) {
 		if (params.key_mgmt_suite == WPA_KEY_MGMT_IEEE8021X ||
 		    params.key_mgmt_suite == WPA_KEY_MGMT_IEEE8021X_SHA256 ||
